configure.ac

```diff
@@ -1,6 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 #AC_PREREQ([2.62])
 
+# This line is parsed by mono.winconfig.targets.
```

Comment:
"other tools (besides autoconf)" agreed. The format is pretty flexible though, so not sure what to say what to leave alone vs. what is changeable. More like, you can search them out and decide, or indeed, you can stick to what the strict format appears to be.

In this case, you know, the format was changeable between uppercase and lowercase, and the parsing was bad.

Reply:
Why do we need all this stuff ? Can't we just generate the guid using the perl line above ?

Reply:
Which stuff? The .targets in the first place? Or whatever else reads the file? .targets because, msbuild & no autoconf/bash.  Other readers I don’t know. The generated guid tends to be lowercase. The msbuild code was requiring uppercase, and a guid. It should just take a string. Limiting it to a guid was overkill and requiring uppercase was failing.

Reply:
If you want, we can insist on a guid, but it seems unnecessary. The minimum fix is to allow lowercase. It also had a bad failure mode but requirements there are subjective.

Reply:
I reworded the comments. Perhaps they say too much though.